support-frontend/assets/components/dialog/dialog.tsx

```diff
@@ -23,10 +65,7 @@ function Dialog({
 }: PropTypes): JSX.Element {
 	return (
 		<div
-			className={classNameWithModifiers('component-dialog', [
-				open ? 'open' : null,
-				styled ? 'styled' : null,
-			])}
+			css={[dialog, open && openCss, styled && styledCss]}
```

Comment:
Havent used that method before, saves passing the parameter in side the css, great